apps/client/src/pages/QuestionsPage.js

```diff
@@ -46,34 +50,78 @@ const QuestionsPage = ({ topic, checker }) => {
     });
 
   const onQuestionNext = (value) => {
-    if (question.options) {
+    // Provide the user answers to the `sttr-checker`
+    if (question.options && value) {
```

Comment:
Can you be a bit more explicit and check answers if the are not `undefined`?

Reply:
`if (question.options && value !== undefined) {` ?
Or shall we move this to a different ticket?

Reply:
Replacing this with value !== undefined is working. Marking as resolved